### DISCUSSION TITLE
New to Security Onion

### DISCUSSION THREAD

**Discussion by drmanhattan8** [Security-Onion-Solutions/securityonion#4296] [Unsupported Versions]
Hi All, 

New to SO, sorry.  I am trying to install RITA and it only installs on the manager node.  How can I use RITA on the manager node to evaluate beacons.  Sorry for my noob questions.

Thank you,
DrM

**Comment by presianbg** [comment]
I assume you have distributed setup, if so:

You have to install RITA on every sensor as no Zeek logs being stored on the Search / Manager nodes.
